npm/ux/histogram.tsx

```diff
@@ -242,218 +244,195 @@ export function Histogram(props: {
   }
 
   return (
-    <svg class="histogram" viewBox="0 0 165 100" onWheel={onWheel}>
-      <g transform={`translate(${scale.offset},4)`}>
-        {bucketArray.map((entry, idx) => {
-          const label = showKetLabels ? resultToKet(entry[0]) : entry[0];
-
-          const height = barAreaHeight * (entry[1] / sizeBiggestBucket);
-          const x = barBoxWidth * idx + barPaddingSize;
-          const labelX = barBoxWidth * idx + barBoxWidth / 2 - fontOffset;
-          const y = barAreaHeight + 15 - height;
-          const barLabel = `${label} at ${(
-            (entry[1] / totalAllBuckets) *
-            100
-          ).toFixed(2)}%`;
-          let barClass = "bar";
-
-          if (entry[0] === props.filter) {
-            barClass += " bar-selected";
-          }
-
-          return (
-            <>
-              <rect
-                class={barClass}
-                x={x}
-                y={y}
-                width={barFillWidth}
-                height={height}
-                onMouseOver={onMouseOverRect}
-                onMouseOut={onMouseOutRect}
-                onClick={onClickRect}
-                data-raw-label={entry[0]}
-              >
-                <title>{barLabel}</title>
-              </rect>
-              {
-                <text
-                  class="bar-label"
-                  x={labelX}
-                  y="85"
-                  visibility={showLabels ? "visible" : "hidden"}
-                  transform={`rotate(90, ${labelX}, 85)`}
+    <>
+      {props.shotsHeader ? (
```

Comment:
This and the next 2 lines are largely the only change in the output here. The rest is mostly whitespace updates.